doc/src/manual/running-external-programs.md

```diff
@@ -373,3 +373,13 @@ stages have different latency so they use a different number of parallel workers
 saturated throughput.
 
 We strongly encourage you to try all these examples to see how they work.
+
+## `Cmd` Objects
+[`Cmd`](@ref) objects can also be used. It can customize them with the `dir` keyword to set the working directory,
```

Comment:
maybe can also be used *directly*, since the cmd syntax does also produce Cmd objects. And then adjust the rest of the sentence to say "This way, they may be used to customize ..."

Reply:
Thanks. I fixed it. PTAL.